engines/windowsaio.c

```diff
@@ -353,6 +429,16 @@ static int fio_windowsaio_getevents(struct thread_data *td, unsigned int min,
 	return dequeued;
 }
 
+static int fio_windowsaio_getevents(struct thread_data *td, unsigned int min,
+				    unsigned int max, const struct timespec *t)
+{
+	struct windowsaio_options *o = td->eo;
+
+	if (o->no_completion_thread)
+    	return fio_windowsaio_getevents_nothread(td, min, max, t);
+	return fio_windowaio_getevents_thread(td, min, max, t);
```

Comment:
Wrong indentation in the first return.

```suggestion
	if (o->no_completion_thread)
		return fio_windowsaio_getevents_nothread(td, min, max, t);
	return fio_windowaio_getevents_thread(td, min, max, t);
```

Reply:
fixed